frontend/src/scenes/paths/NewPaths.js

```diff
@@ -371,11 +374,54 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
                                             textAlign: 'start',
                                         }}
                                     >
-                                        <span
-                                            className="text-muted"
-                                            style={{ fontSize: 10, marginRight: 4, marginLeft: 8 }}
-                                        >{`0${pathItemCard.name[0]}`}</span>{' '}
-                                        <span style={{ fontSize: 13, fontWeight: 600 }}>{pageUrl(pathItemCard)}</span>
+                                        <Row justify="space-between" align="center">
+                                            <div>
+                                                <span
+                                                    className="text-muted"
+                                                    style={{ fontSize: 10, marginRight: 4, marginLeft: 8 }}
+                                                >{`0${pathItemCard.name[0]}`}</span>{' '}
+                                                <span style={{ fontSize: 13, fontWeight: 600 }}>
+                                                    {pageUrl(pathItemCard)}
+                                                </span>
+                                            </div>
+                                            <div>
+                                                <Dropdown
+                                                    trigger={['click']}
+                                                    overlay={
+                                                        <Menu className="paths-options-dropdown">
+                                                            <Menu.Item
+                                                                onClick={() =>
+                                                                    setFilter({ start_point: pageUrl(pathItemCard) })
```

Comment:
I'm guessing this wouldn't work for anything > length 25 ? Since we need the filter to have the exact key. Same below

Reply:
yeah failing example here:

https://user-images.githubusercontent.com/13127476/135468768-34d5b8ae-f1c9-45a0-8d03-b45aba9ba3a0.mov

Reply:
oh ugh yeah great point

Reply:
our exclusions reuse the propertyfilter logic so you need to match the propertyfilter objects like this: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/insights/InsightTabs/PathTab/NewPathTab.tsx#L376

also `updateExclusions` is a "replace all" type function it doesn't just append new exclusions so that needs to be refactored or accounted for

Reply:
so you're saying `updateExclusions` right now is incorrect even on the query builder?

Reply:
it is correct you'll just need to make sure usage matches

Reply:
updated and now exclusion works properly, ideally i'd refactor this since mapping in a kea action is probably not good practice but since we want to get this out for testing we can always do that in the next PR :)